src/overdrive.js

```diff
@@ -4,6 +4,19 @@ import ReactDOM  from 'react-dom'
 const renderSubtreeIntoContainer = ReactDOM.unstable_renderSubtreeIntoContainer;
 const components = {};
 
+const prefix = styles => {
```

Comment:
Can you export to a different file and `import` it? will be easier to test when we do add unit tests